src/App.js

```diff
@@ -1,27 +1,9 @@
-/* eslint-disable */
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
```

Comment:
- [ ] According to the project requirements, The `App.js` component should also be refactored in class-based component  kindly make this, It might look like 👇🏻 

./src/App.js
```
// src/App.js

import React, { Component } from 'react';
import Calculator from ./components; // Import Calculator component

class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```